File: dataset/codenet/Java/Code/s545439898.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args){
		
		Scanner sc = new Scanner(System.in);
		int count = Integer.parseInt(sc.next());
		int[] givingNumbers = new int[count];
		
		// Read Numbers
		for(int i = 0; i< count;i++){
			
			givingNumbers[i] = Integer.parseInt(sc.next());
			
		}
		
		// Reversing Numbers
		for(int i = count-1; 0 <= i;i--){
			
			System.out.print(givingNumbers[i]);
			
			if(i > 0){
				System.out.print(" ");
			}
			
			
		}
		
		System.out.println();
		
		sc.close();
	}

}

File: output/qwen_2.5_72b_instruct/codenet/direct_translation/java/python/failed_translation/s545439898.py

sc = input().strip()
count = int(sc)
givingNumbers = []

for i in range(count):
    givingNumbers.append(int(input().strip()))

for i in range(count-1, -1, -1):
    print(givingNumbers[i], end=" " if i > 0 else "\n")

